src/nanoemoji/color_glyph.py

```diff
@@ -313,10 +313,10 @@ class ColorGlyph(NamedTuple):
     glyph_id: int
     codepoints: Tuple[int, ...]
     painted_layers: Tuple[PaintedLayer, ...]
-    picosvg: SVG
+    svg: SVG  # picosvg except for untouched formats
 
     @staticmethod
-    def create(ufo, filename, glyph_id, codepoints, picosvg):
+    def create(ufo, filename, glyph_id, codepoints, svg, extract_layers=True):
```

Comment:
this commit does a lot more than simply "adding tests for picosvg and untouchedsvg formats"

Reply:
I named it that specifically so you wouldn't notice! In retrospect I probably should have tidied up the commits and comments.